.taskcluster.yml

```diff
@@ -130,6 +132,10 @@ tasks:
                                 then:
                                     - "tc-treeherder.v2.${project}.${head_sha}"
                                 else: []
+                              - $if: 'tasks_for == "github-push"'
+                                then:
+                                    - "index.${trustDomain}.v2.${project}.revision.${head_sha}.taskgraph.decision"
```

Comment:
Question: Do we want to add all possibles routes? I blindly added them in Fenix, but I'm not sure if it's worth the effort.

https://github.com/mozilla-mobile/fenix/blob/3d387e2dde9bf2f0265c9467a1d9e04c683c3144/.taskcluster.yml#L169-L178

Reply:
Sure. This is the one the action needed, but I might as well.

Reply:
We do have the problem of `project.mobile.v2` vs `mobile.v2`. It looks like Fenix has the former, and taskgraph has [the latter](https://hg.mozilla.org/ci/taskgraph/file/tip/src/taskgraph/util/taskgraph.py#l28). I'll copy the fenix indexes, but remove `project.`. We can discuss here before landing anything.

Reply:
I know `project.` is legacy. It dates back when everything was hosted on tools.taskcluster.net. @tomprince wanted to move mobile projects outside of `project.`, so it could be a good time to do it. 

That said, I'm wondering if having 2 different prefixes won't eventually bite us. 

I'm fine landing this as is and see what happens later.

Reply:
+1. This probably means we should standardize, which probably means we should:

1. add non-`project.` indexes to Fenix
2. mark the `project.` indexes as deprecated, and cut over usage of the indexes to the non-`project.` indexes
3. remove the `project.` indexes